rush.json

```diff
@@ -345,6 +324,11 @@
       "projectFolder": "packages/example-app-base",
       "versionPolicyName": "lockedMajor"
     },
+    {
+      "packageName": "@uifabric/theming-core",
+      "projectFolder": "packages/theming-core",
+      "versionPolicyName": "lockedMajor"
```

Comment:
Specifying `versionPolicyName` before this package is manually published will cause some issues. I can help you manually publish this package before we merge this PR.